src/ostree/ot-builtin-refs.c

```diff
@@ -43,7 +43,6 @@ static gboolean do_ref (OstreeRepo *repo, const char *refspec_prefix, GCancellab
   GHashTableIter hashiter;
   gpointer hashkey, hashvalue;
   gboolean ret = FALSE;
-  OstreeRepoTransactionStats stats;
   g_autofree char *parent = NULL;
```

Comment:
Yeah, that's the cost of using `g_autofree`. It makes GCC think that it's used even when it's not (though technically it is... in the cleanup function).